jans-auth-server/model/src/main/java/io/jans/as/model/token/JsonWebResponse.java

```diff
@@ -22,8 +22,8 @@
 
     private static final long serialVersionUID = -4141298937204111173L;
 
-    protected JwtHeader header;
-    protected JwtClaims claims;
+    protected transient JwtHeader header;
+    protected transient JwtClaims claims;
```

Comment:
I have implemented writeObject and readObject to allow serialization. 
It was because Map<String,Object> in JwtClaimSet (parent) is not serializable.
I have implemented one unit test too, to check serialization.